### DISCUSSION TITLE
Include more files into release versions: Not only executables, but also libraries

### DISCUSSION THREAD

**Discussion by kfriedberger** [cvc5/cvc5#9678] [General]
As a developer of another tool, I would like to download complete prepared/compiled release versions for CVC5 (executables and libraries, including all dependencies, bindings, and license information).
Currently, only the executables are [available there](https://github.com/cvc5/cvc5/releases),
which requires us to build the whole CVC5 aplication on our own, using our own flags and handling potential errors coming from the build-process.

An optimal solution can be seen at the [Z3 release page](https://github.com/Z3Prover/z3/releases), where there is a ZIP-file per operating systems, each including all bindings and libraries for further development.
I would assume that such files already come out from the default build process for CVC5, and only the packaging ignores them.
Would it be possible to provide such files also at the CVC5 release page?

**Comment by vrcamillo** [comment]
Library packages for Linux and Windows are uploaded [here](https://github.com/cvc5/cvc5/releases/tag/cvc5-1.0.5).
